contrib/database/sql/conn.go

```diff
@@ -167,6 +211,48 @@ func WithSpanTags(ctx context.Context, tags map[string]string) context.Context {
 	return context.WithValue(ctx, spanTagsKey, tags)
 }
 
+// tryStartTrace will create a span using the given arguments, but will act as a no-op when err is driver.ErrSkip.
```

Comment:
This comment mentions it's a no-op for driver.ErrSkip but that's not true currently (I guess this will probably change based on the other comment thread)